recipes/devel/rustc.yaml

```diff
@@ -0,0 +1,155 @@
+inherit: [install]
+
+metaEnvironment:
+    PKG_VERSION: "1.81.0"
+    PKG_LICENSE: "Apache-2.0 or MIT"
+
+privateEnvironment:
+    RUST_MIRROR: "https://static.rust-lang.org"
+
+multiPackage:
+    # rust compiler + std lib + tools like cargo
+    "":
+        # provide the rustc package based on the selected target architecture
+        depends:
+            - "devel::rustc-$(get-tool-env,target-toolchain,ARCH)"
```

Comment:
In the long run, `$ARCH` is certainly not sufficient. But that doesn't mean we must already roll the generic solution either. So until other OSes are supported, we might want to name the variants accordingly, e.g. `devel::rustc-x86_64-unknown-linux-gnu` -> `"devel::rustc-$(get-tool-env,target-toolchain,ARCH)-unknown-linux-gnu"`. This would kill the arm variant but it is better than requiring to refactor other recipes in the long run...

Reply:
This would also kill the aarch64 variant (which I need). Should I try to use some ifelse magic or is it really worth that effort?

Reply:
Actually it would also kill the i386 variant ;) I would prefer to wait until we exactly know how this would look like for different OS's.

Reply:
Well, it removed some functionality when installing rust on the target. On the previous change you could install the rust compiler on the target + any extra std libs you want for doing cross compilation on the target. This is not possible anymore. Also are those if expr in the depends section possible in the current version of bob?

Reply:
Sounds ok to me.